### DISCUSSION TITLE
Synology DSM 7.2.2-72806 Update 2

### DISCUSSION THREAD

**Discussion by Samurai97** [TheAxelander/OpenBudgeteer#276] [Q&A] [answered]
Since installing this update of DSM I am now unable to restart my OpenBudgeteer instance as it continually says it is "waiting for database". The database has been created in MariaDB 10 which is a package based database install and is therefore external to the OpenBudgeteer container. 

I have dropped and recreated the container but with not success.

Script as follows:

docker run -d --name='openbudgeteer' \
    -e 'CONNECTION_PROVIDER'='mariadb' \
    -e 'CONNECTION_SERVER'='nnn.nnn.nnn.nnn' \
    -e 'CONNECTION_PORT'='nnnn' \
    -e 'CONNECTION_DATABASE'='OpenBudgeteer' \
    -e 'CONNECTION_USER'='OpenBudgeteer' \
    -e 'CONNECTION_PASSWORD'='xxxxxxxxxxxxxxxx' \
    -e 'MYSQL_ROOT_PASSWORD'='xxxxxxxxxx' \
    -e 'APPSETTINGS_CULTURE'='en-GB' \
    -p 'nnnn:8080' \
    -v /volume2/docker/openbudgeteer:/app/database \
    --restart=always \
    --add-host=localhost:nnn.nnn.nnn.nnnn \
    'axelander/openbudgeteer'

With a docker instance via the Synology Container Manager, is there a way to get more detailed log messages to determine cause of failure?

**Comment by Samurai97** [comment]
Oh for info, the Synology release notes for this update are very helpful !!!

"Minor bug fixes" !!

**Comment by TheAxelander** [comment]
If it says "waiting for database" for a long time it's more or less a sign, that the database container is no longer accessible for whatever reason. 

So I could also only guess the reason for it. Maybe the different container now run in isolated networks, but yeah not much where I can help with unfortunately.

**Reply by Samurai97** [reply, marked as answer]
OK I figured it out, it was a change in my firewall rules.  I forgot that in order to link from the OpenBudgeteer container to my "local" network holding the MariaDB container, I had to permit connections from the container bridge network IP address, rather than just my network sub-net.
